src/components/sidebar/SafeList/index.tsx

```diff
@@ -59,13 +64,55 @@ export const _shouldExpandSafeList = ({
 const MAX_EXPANDED_SAFES = 3
 const NO_SAFE_MESSAGE = 'Create a new safe or add'
 
+export const addActionsToTxs = (
```

Comment:
I don't understand from the name what the function does and why it manipulates some transactions in the Safe List.
The body of the function doesn't help either. Some comments would be helpful, or better, a refactoring to simplify it.

Reply:
The function calculates 2 values:
- the transactions that can be signed
- the size of the transaction queue

having being passed a `TransactionListPage` to it and the object that we are building `txsActionsByChain`.

Then it returns an object
```
{
  [chainId]: {
    [safeAddress]: {
      signing?: <number | string>
      queued?: <number | string>
    }
  }
}
```

to later be passed to each SafeListItem.

I am about to rename the function to `addActionsToSafeTxs` but WDYT? Any suggestion?